.github/ISSUE_TEMPLATE/bug.yml

```
@@ -25,6 +25,7 @@ body:
       options:
         - "Yes"
         - "No"
+      default : 0
```

```suggestion
      default : 1 
```
The default option should be 1 which will default the value to No.

---

In the example template I provided, I switched the yes and no around, so it can default to `0`.

@Sathasivamthirumoorthi please use the example in the issue